doc/release_checklist.md

```diff
@@ -0,0 +1,38 @@
+1. Change version number in
+   - /configure.ac
+   - /CMakeLists.txt
+   - /doc/Doxyfile.in
+   - /man/flac.md
+   - /man/metaflac.md
+   - /test/metaflac-test-files/case07-expect.meta
+1. Change version date in
```

Comment:
All items are number 1 ? :)

Reply:
Yes, when doing that in markdown, it renders properly on github. That way one doesn't have to change the numbering on inserting an item. See [here](https://github.com/xiph/flac/blob/923bd32633cd3667ba404cdf3d24e4e515338668/doc/release_checklist.md)

Reply:
Huh. Learnt something new..